### PR TITLE
RILENA 2

### PR DESCRIPTION
**Pull request by VernaSaurus**
## About The Pull Request

Added a bunch more RILENA stuff!
FOOD:
image
PLUSHIES:
image
POSTERS:
image

## Why It's Good For The Game

MORE RILENA

## Changelog

:cl:
add: Added a bunch more RILENA stuff! Five more posters, three more plushies, a popsicle, some macaroni and cheese, and, of course, the elusive RILENA MRE.
/:cl:

### CODE REVIEW

File: code/modules/food_and_drinks/food/ration.dm [hunk 1]
```diff
@@ -285,6 +285,14 @@
 	tastes = list("cheese" = 1, "pizza" = 1)
 	foodtype = GRAIN | DAIRY
 
+/obj/item/reagent_containers/food/snacks/ration/entree/cereal
+	name = "/improper RILENA flavored cereal"
```
Comment (MarkSuckerberg) [inline code/modules/food_and_drinks/food/ration.dm:289]:
```suggestion
	name = "\improper RILENA flavored cereal"
```

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]

**Comment by meemofcourse** [comment]
fix ur conflicts and address review pls

**Comment by thgvr** [comment]
I think we should probably go a little less hard on this stuff

**Comment by thgvr** [comment]
No activity in a while, ping me if you'd like to work on this again

**Pull request closed without merging**